src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -31,12 +31,12 @@
  * on separate line from target element.
  * <p>
  * Attention: Annotations among modifiers are ignored (looks like false-negative)
- * as there might be a problem with annotations for return types
+ * as there might be a problem with annotations for return types.
  * </p>
  * <pre>public @Nullable Long getStartTimeOrNull() { ... }</pre>.
  * <p>
  * Such annotations are better to keep close to type.
- * Due to limitations Checkstyle can not examin target of annotation.
+ * Due to limitations Checkstyle can not examin the target of annotation.
```

Comment:
`annotation` -> `the annotation` or `an annotation`
I am not sure, @rnveach can you take a look please?